### PR TITLE
Adding CheckoutDays to xsd

### PR DESCRIPTION
**Pull request by lolamastro**

### CODE REVIEW

File: xml-specification/property-manager-data-feeds-1.1.xsd [hunk 1]
```diff
@@ -94,6 +94,7 @@
                         <xsd:element minOccurs="0" name="MonthlyMaxRate" type="money" />
                         <xsd:element minOccurs="0" name="MinimumStayLength" type="xsd:int" />
                         <xsd:element minOccurs="0" name="TurnDay" type="xsd:string" />
+                        <xsd:element minOccurs="0" name="CheckoutDays" type="CheckoutDayList" default="Sunday Monday Tuesday Wednesday Thursday Friday Saturday"/>
```
Comment (rynemccall) [inline xml-specification/property-manager-data-feeds-1.1.xsd]: Though we don't do this for the other similar elements, I think you should add maxOccurs="1".

File: xml-specification/property-manager-data-feeds-1.1.xsd [hunk 2]
```diff
@@ -276,4 +277,19 @@
...
+            <xsd:enumeration value="Tuesday"/>
+            <xsd:enumeration value="Wednesday"/>
+            <xsd:enumeration value="Thursday"/>
+            <xsd:enumeration value="Friday"/>
+            <xsd:enumeration value="Saturday"/>
+        </xsd:restriction>
+    </xsd:simpleType>
+
+    <xsd:simpleType name="CheckoutDayList">
+        <xsd:list itemType="DayOfWeek" />
```
Comment (rynemccall) [inline xml-specification/property-manager-data-feeds-1.1.xsd]: I think this should be minOccurs="1" maxOccurs="7" and require that DayOfWeek element be unique.

File: xml-specification/property-manager-data-feeds-1.1.xsd (hunk 1 above)
Comment (lolamastro) [reply xml-specification/property-manager-data-feeds-1.1.xsd]: done

File: xml-specification/property-manager-data-feeds-1.1.xsd (hunk 2 above)
Comment (lolamastro) [reply xml-specification/property-manager-data-feeds-1.1.xsd]: Did the min and max but don't know the unique syntax given its a list of strings.  Do you know?

**Comment by rynemccall** [comment]
It's probably not worth our time to track down how to enforce distinct days of weeks, especially in a spec this old. I'll merge the PR.

**Pull request merged**